### PR TITLE
documentation(926930): Added getting-started for chat ui.

### PR DESCRIPTION
**Pull request by VigneshwaranGovindharajan**
### Description

Need to add the Getting started for Chat UI Control.
  
### Screenshot

** Default **

chat-ui-component

** Default Messages **

image

### Areas affected and ensured

None

### Test cases

NA

### Test bed sample location

NA

### Additional checklist

* Did you run the automation against your implementation? NA
* Is there any API name change? NA.
* Is there any existing behavior change of other features due to this code change? No
* Does your new code introduce new warnings or binding errors? No
* Does your code pass all FxCop and StyleCop rules? No
* Did you record this case in the unit test or UI test? No

### CODE REVIEW

File: ej2-asp-core-mvc/chat-ui/EJ2_ASP.NETCORE/getting-started.md [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+
+You can use the `e-chatui-messages` tag directive to add messages and the `user` tag directive to map each message to the respective user.
+
+{% tabs %}
+{% highlight cshtml tabtitle="CSHTML" %}
+{% include code-snippet/chat-ui/default-messages/tagHelper %}
+{% endhighlight %}
+{% endtabs %}
+
+![ASP.NET Core AI AssistView default messages](images/default-messages.png)
```
Comment (IndrajithSrinivasan) [inline ej2-asp-core-mvc/chat-ui/EJ2_ASP.NETCORE/getting-started.md:112]: why used assistview component name @VigneshwaranGovindharajan

File: ej2-asp-mvc-toc.html [hunk 2]
```diff
@@ -529,6 +529,12 @@
 				<li><a href="/cr/aspnetmvc-js2/Syncfusion.EJ2.Charts.CircularChart3D.html">API Reference</a></li>
 			</ul>
 		</li>
+
+		<li>ChatUI
```
Comment (IndrajithSrinivasan) [inline ej2-asp-mvc-toc.html:533]: space not added here @VigneshwaranGovindharajan  **Chat UI**

File: ej2-asp-core-mvc/chat-ui/EJ2_ASP.MVC/getting-started.md [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+{% tabs %}
+{% highlight razor tabtitle="CSHTML" %}
+{% include code-snippet/chat-ui/default-messages/razor %}
+{% endhighlight %}
+{% highlight c# tabtitle="Default.cs" %}
+{% include code-snippet/chat-ui/default-messages/defaultMessages.cs %}
+{% endhighlight %}
+{% endtabs %}
+
+![ASP.NET MVC AI AssistView default messages](images/default-messages.png)
```
[the remark this answers is not in the document]
Comment (VigneshwaranGovindharajan) [reply ej2-asp-core-mvc/chat-ui/EJ2_ASP.MVC/getting-started.md]: Changed control name.

File: ej2-asp-core-mvc/chat-ui/EJ2_ASP.NETCORE/getting-started.md (hunk 1 above)
Comment (VigneshwaranGovindharajan) [reply ej2-asp-core-mvc/chat-ui/EJ2_ASP.NETCORE/getting-started.md]: Changed control name.

File: ej2-asp-mvc-toc.html (hunk 2 above)
Comment (VigneshwaranGovindharajan) [reply ej2-asp-mvc-toc.html]: Updated as per comments.

**Review by IndrajithSrinivasan — APPROVED** [review]

**Pull request merged**